### PR TITLE
add connection with a proxy

### PR DESCRIPTION
**Pull request by DanProgs**
Pull request for add connectionwith a proxy

### CODE REVIEW

File: src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java [hunk 1]
```diff
@@ -148,6 +149,37 @@ public TestLinkAPI(URL url, String devKey) throws TestLinkAPIException {
 
         this.miscService.checkDevKey(devKey);
     }
+    
+    // add connection with a proxy
```
Comment (kinow) [inline src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java]: Ah, just had time to see the error in Travis. @DanProgs could you add a Javadoc here, and check the trailing spaces here:

```
[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[152] (regexp) RegexpSingleline: Line has trailing spaces.

[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[154,5] (javadoc) JavadocMethod: Missing a Javadoc comment.

[ERROR] src/main/java/br/eti/kinoshita/testlinkjavaapi/TestLinkAPI.java:[159] (regexp) RegexpSingleline: Line has trailing spaces.
```

**Comment by DanProgs** [comment]
Hi there,
I added the Javadoc and removed a few blank lines. I hope it's a little better now?

**Comment by kinow** [comment]
Travis CI is green now, LGTM. Thanks @DanProgs !

**Comment by kinow** [comment]
Released 1.9.20-0 just now (might take a while for it to sync with Maven central). Thanks for the PR!

**Pull request merged**